src/main/java/com/zendesk/maxwell/MaxwellMysqlConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.zendesk.maxwell;
+
+import joptsimple.OptionSet;
+
+/**
+ * Created by kaufmannkr on 12/23/15.
+ */
+public class MaxwellMysqlConfig {
+
+	public String mysqlHost;
+	public Integer mysqlPort;
+	public String mysqlUser;
+	public String mysqlPassword;
```

Comment:
no need to reiterate "mysql" here I think.  These fields can be just "host, port, user, password"